pkg/storage/storage.go

```diff
@@ -809,6 +810,12 @@ func (is *ImageStore) FinishBlobUpload(repo string, uuid string, body io.Reader,
 		}
 	}
 
+	err = chattr.AddImmutable(src)
```

Comment:
Looks like this library just execs `chattr`, if we're going to use a library it might be nice to use one that actually does the ioctls in case someone doesn't have e2fsprogs installed. Otherwise we can just exec the chattr ourselves and avoid the dep.

Reply:
It also just fails silently if chattr isn't present :\

https://github.com/libopenstorage/openstorage/blob/master/pkg/chattr/chattr.go#L19

Reply:
I see, this one seems to use ioctls: https://github.com/g0rbe/go-chattr/blob/master/chattr.go
Is this one ok?

Reply:
Yeah, that looks good to me, thanks!